src/lib/protocols/netbios.c

```diff
@@ -80,7 +80,8 @@ static void ndpi_int_netbios_add_connection(struct ndpi_detection_module_struct
   char name[64];
   u_int off = flow->packet.payload[12] == 0x20 ? 12 : 14;
   
-  if(ndpi_netbios_name_interpret((char*)&flow->packet.payload[off], flow->packet.payload_packet_len - off, name, sizeof(name)) > 0)
+  if(off > flow->packet.payload_packet_len &&
```

Comment:
This is a bug it should be (off < flow->.....)

Reply:
Indeed, just force pushed the fix

Would you like me to open a new PR on this ?
Or do you fix it yourself ?

Reply:
Thanks for already fixing this